portal-impl/src/com/liferay/portlet/documentlibrary/service/impl/DLFileEntryLocalServiceImpl.java

```diff
@@ -2552,12 +2553,11 @@ protected DLFileVersion updateFileVersion(
 	}
 
 	protected void validateFile(
-			long groupId, long folderId, long fileEntryId,
-			String sourceFileName, String fileName, String extension,
-			String title)
+			long groupId, long folderId, long fileEntryId, String fileName,
+			String extension, String title)
 		throws PortalException {
 
-		DLValidatorUtil.validateFileName(title);
+		DLValidatorUtil.validateFileName(fileName);
```

Comment:
With this yo do not validate the title, so "illegal" characters are permitted now. 
There is a pending story for this that implies to talk with Pablo.

Reply:
Ok we can talk to Pablo about it. I personally think title should allow anything, or there should be a specific `validateTitle` function.

Reply:
The problem with this, now is that it can this is a conflict with https://issues.liferay.com/browse/LPS-123544 
So, yes, we have to talk with him to this issue : https://issues.liferay.com/browse/LPS-122738